From a308aa5dd9adfa4c534da380444991c33a6c7d0d Mon Sep 17 00:00:00 2001
Message-Id: <a308aa5dd9adfa4c534da380444991c33a6c7d0d.1660490052.git.git@grubix.eu>
From: Michael J Gruber <git@grubix.eu>
Date: Sun, 14 Aug 2022 16:36:38 +0200
Subject: [PATCH] test: increase cffi timeout

By default, the test suite uses 2min for other tests and 5s for cffi
tests. Sporadically, this leads to test failures caused by the timeout
on slower or loaded test infrastructure (as seen on ppc64le in Fedora's
infrastructure during branch time).

Increase the cffi timeout to the same 2m=120s.

Signed-off-by: Michael J Gruber <git@grubix.eu>
---
 bindings/python-cffi/tests/conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bindings/python-cffi/tests/conftest.py b/bindings/python-cffi/tests/conftest.py
index 6835fd30..fe90c787 100644
--- a/bindings/python-cffi/tests/conftest.py
+++ b/bindings/python-cffi/tests/conftest.py
@@ -43,7 +43,7 @@ def notmuch(maildir):
         env = os.environ.copy()
         env['NOTMUCH_CONFIG'] = str(cfg_fname)
         proc = subprocess.run(cmd,
-                              timeout=5,
+                              timeout=120,
                               env=env)
         proc.check_returncode()
     return run
-- 
2.37.2.596.g72ccb331cf

